back-end/__tests__/endpoints.test.js

```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
+  it('Testing that endpoints are posting data', () => {
```

Comment:
Ideally, tests should read as if it were in English.
* "it testing that endpoints are posting data" => not valid English
* "it tests that endpoints are posting data" => valid English